plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJmxStats.java

```diff
@@ -44,14 +43,14 @@ public void testJmxStatsExposure()
         MBeanServer mbeanServer = getPlatformMBeanServer();
         Set<ObjectName> objectNames = mbeanServer.queryNames(new ObjectName("io.trino.plugin.jdbc:*"), null);
 
-        assertTrue(objectNames.containsAll(
+        assertThat(objectNames.containsAll(
                 ImmutableSet.of(
                         new ObjectName("io.trino.plugin.jdbc:type=ConnectionFactory,name=test"),
-                        new ObjectName("io.trino.plugin.jdbc:type=JdbcClient,name=test"))));
+                        new ObjectName("io.trino.plugin.jdbc:type=JdbcClient,name=test")))).isTrue();
```

Comment:
Formatting nit:

```suggestion
                        new ObjectName("io.trino.plugin.jdbc:type=JdbcClient,name=test"))))
                .isTrue();
```

(It could be improved in general)